app/views/schools/ects/show.html.erb

```diff
@@ -9,69 +9,14 @@
 
 <%= render Schools::TeacherProfileSummaryListComponent.new(@ect) %>
 
-<h2 class='govuk-heading-m'>ECTE training details</h2>
-<%=
-  govuk_summary_card(title: 'Reported to us by your school') do |card|
-    if @ect.provider_led?
-      card.with_summary_list(
-        rows: [
-          { key: { text: 'Appropriate body' },
-            value: { text: appropriate_body_name(appropriate_body_type: @ect.appropriate_body_type,
-                                                 appropriate_body: @ect.appropriate_body) } },
-          { key: { text: 'Programme type' },   value: { text: programme_type_name(@ect.programme_type) } },
-          { key: { text: 'Lead provider' },    value: { text: @ect.lead_provider_name } },
-        ]
-      )
-    else
-      card.with_summary_list(
-        rows: [
-          { key: { text: 'Appropriate body' },
-            value: { text:appropriate_body_name(appropriate_body_type: @ect.appropriate_body_type,
-                                                appropriate_body: @ect.appropriate_body) } },
-          { key: { text: 'Programme type' },   value: { text: programme_type_name(@ect.programme_type)  } },
-        ]
-      )
-    end
-  end
-%>
+<h2 class="govuk-heading-m">ECTE training details</h2>
```

Comment:
I propose that we keep this out of the component and move [this one](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/components/schools/teacher_profile_summary_list_component.rb#L21) out of the component too.

The goal is to make the page easier to read and understand without needing to look inside the components. By keeping structural elements like headings in the view, it’s clearer how the page is laid out at a glance.

What are your thoughts on that @peteryates?

Reply:
I kind of assume that if we're rendering just a group of components on a page it makes the structure clear, but I see your point.

What about

```erb
<%= render SomeComponent(header: "Widgets", other_things: @stuff)
```

Reply:
Hmm, I would have liked: 

`<%= render SomeComponent(h2: "H2", heading: 'component heading', other_things: @stuff)`

But i'll keep it out of the component because we're now using that same component to render three of the summary cards but the same `h2` is only shown once at the top. 

I think this approach keeps it simple.

Thank you!